python_modules/dagster/dagster/core/storage/event_log/sql_event_log.py

```diff
@@ -153,8 +163,9 @@ def store_asset_materialization(self, event):
         #
         # https://github.com/dagster-io/dagster/issues/3945
 
-        # last_materialization_timestamp is updated upon observation or materialization
-        # See store_asset method above for more details
+        # last_materialization_timestamp is updated upon observation, materialization, materialization_planned
+        # last_run_id column is updated upon materialization and materialization planned events.
```

Comment:
Not sure if it's redundant to continue to update `last_run_id` upon materialization and materialization planned, but kept it for now